symplyphysics/laws/chemistry/cross_section_of_interaction_in_elastic_interaction_model.py

```diff
@@ -15,8 +15,8 @@
 cross_sectional_area_of_interaction = Symbol("cross_sectional_area_of_interaction", units.area)
 
 diameter_of_atom = Symbol("diameter_of_atom", units.length)
-constant_sutherland = Symbol("constant_sutherland", units.temperature)
-temperature = Symbol("temperature", units.temperature)
+constant_sutherland = clone_symbol(symbols.thermodynamics.temperature, "constant_sutherland")
```

Comment:
I don't think it is essentially a temperature, as in global symbols. Don't clone it.